tools/devops/automation/templates/windows/stage.yml

```diff
@@ -24,6 +24,14 @@ parameters:
 - name: isPR
   type: boolean
 
+- name: repositoryAlias
```

Comment:
There's no way to set an environment variable (or some other to pass data) for the entry template, and then load that value whenever needed, instead of adding more and more template parameters?

You're adding 350 lines of code to use a value 6 times...

Reply:
unfortunately, there is not since it is a template and the template cannot use variables that are not the default ones.